python/raft-ann-bench/src/raft-ann-bench/run/__main__.py

```diff
@@ -574,8 +575,8 @@ def add_algo_group(group_list):
                         index["search_params"].append(search_dict)
                 executables_to_run[executable]["index"].append(index)
 
-            if len(index["search_params"]) == 0:
-                print("No search parameters were added to configuration")
+                if len(index["search_params"]) == 0:
+                    print("No search parameters were added to configuration")
```

Comment:
Will the C++ benchmark raise an error if search params are empty?

Reply:
I don't think so?

Reply:
Okay, as long as they don't then it's good. Otherwise we should be promoting the error to Python.